### PR TITLE
BUGFIX: RAIL-1232 embedded ad drill event is not fired in pivot table

### PR DESCRIPTION
**Pull request by tomasreichmann**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2325/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1744/ : SUCCESS in 1m 44s

File: src/components/core/PivotTable.tsx [hunk 1]
```diff
@@ -525,6 +526,14 @@ export class PivotTableInner extends
         };
 
         if (onFiredDrillEvent(drillEvent)) {
+            // This is neaded for /analyze/embedded/ drilling with post message
```
Comment (lhorak) [inline src/components/core/PivotTable.tsx]: needed  :)

**Comment by lhorak** [comment]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/499/ : SUCCESS in 19s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1776/ : SUCCESS in 3m 07s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/936/ : UNSTABLE in 16m 54s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/450/ : SUCCESS in 22m 54s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/501/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1779/ : SUCCESS in 2m 12s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/939/ : SUCCESS in 13m 26s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/452/ : SUCCESS in 18m 24s

**Pull request merged**